packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -26,16 +30,35 @@ import {
 } from './TokenSearchUtils'
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
 import { useBalance } from '../../hooks/useBalance'
-import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import { useUSDCWithdrawalConfirmationDialogStore } from './TransferPanel'
+import { ERC20BridgeToken, TokenType } from '../../hooks/arbTokenBridge.types'
 import { useTokenLists } from '../../hooks/useTokenLists'
 import { warningToast } from '../common/atoms/Toast'
 import { TokenRow } from './TokenRow'
+import { CommonAddress } from '../../util/CommonAddressUtils'
+import { ArbOneNativeUSDC } from '../../util/L2NativeUtils'
+import { isNetwork } from '../../util/networks'
 
 enum Panel {
   TOKENS,
   LISTS
 }
 
+const ARB_ONE_NATIVE_USDC_TOKEN = {
+  ...ArbOneNativeUSDC,
+  listIds: new Set<number>(),
+  type: TokenType.ERC20,
```

Comment:
`ARB_GOERLI_NATIVE_USDC_TOKEN` includes an `address` field, but this one is missing it?

Reply:
`ARB_ONE_NATIVE_USDC_TOKEN` has address
I'll take the address out to make it more explicit